var/spack/repos/builtin/packages/psalg/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Psalg(CMakePackage):
+    """LCLS II Developement: PSAlg."""
+
+    homepage = "https://github.com/slac-lcls/lcls2"
+    url = "https://github.com/slac-lcls/lcls2/archive/refs/tags/3.3.37.tar.gz"
+
+    maintainers = ["valmar"]
```

Comment:
@valmar Are you okay with being listed as a maintainer on this package?  See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers for more information.

Reply:
@valmar and I have corresponded by email and it is OK.

Reply:
I apologize for the delay in reacting to this. I am OK being listed as maintainer of psalg